packages/database_v2/artifacts/provision.sql

```diff
@@ -0,0 +1,216 @@
+-- CreateEnum
+CREATE TYPE "resource_status" AS ENUM ('Queued', 'URIParseFailed', 'ContentFetchFailed', 'PinRequestFailed', 'ContentLinked');
+
+-- CreateEnum
+CREATE TYPE "nft_asset_status" AS ENUM ('Queued', 'URIParseFailed', 'ContentFetchFailed', 'ContentParseFailed', 'PinRequestFailed', 'Linked');
+
+-- CreateEnum
+CREATE TYPE "pin_status" AS ENUM ('ClusterError', 'PinError', 'PinQueued', 'Pinned', 'Pinning', 'Remote', 'Sharded', 'Undefined', 'UnpinError', 'UnpinQueued', 'Unpinned', 'Unpinning');
+
+-- CreateEnum
+CREATE TYPE "pin_service" AS ENUM ('Pinata', 'IpfsCluster');
+
+-- CreateTable
+CREATE TABLE "block" (
+    "hash" TEXT NOT NULL,
+    "number" INTEGER NOT NULL,
```

Comment:
In fauna capturing timestamps made little sense as db itself tracked it natively and you could query things in time. In postgres world we'd want to do it manually.

```suggestion
    "number" INTEGER NOT NULL,
    "inserted_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
    "updated_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```